javalang/tokenizer.py

```diff
@@ -128,12 +130,25 @@ def is_postfix(self):
     def is_assignment(self):
         return self.value in self.ASSIGNMENT
 
+    def is_lambda(self):
+        return self.value in self.LAMBDA
+
+
+class MethodReference(JavaToken):
```

Comment:
Lets just add '::' to the set of operators above and let the existing try_operator code handle this case.